docs/guides/views.rst

```diff
@@ -295,6 +295,34 @@ If your entity list will display the entity owners, you can improve performance
 
 See also :doc:`check this page out first </design/database>`.
 
+Since 1.11, you can define an alternative view to render list items using ```'item_view'``` parameter.
```

Comment:
I think "As of {version}" and "As of {date}" type statements are OK, because that information will never go out of date, but might still be useful to someone reading the latest documentation (e.g. if their version of Elgg is not updated to the latest docs).